docker/README.md

```diff
@@ -44,16 +44,16 @@ docker pull dukerobotics/robosub-ros
     
     * Local container
     ```bash
-    ssh -XY -p 2200 duke@localhost
+    ssh -XY -p 2200 root@localhost
```

Comment:
For onboard container (port 2200), remove -XY, since graphics forwarding doesn't work on onboard container.